docs/specs/git.yml

```diff
@@ -1,263 +1,341 @@
 ---
 # Show git contribution info when userProfile & commitsTime is set to URL
-repo: https://github.com/docascode/docfx-test-dependencies#github-user-cache-url
+repos:
+  https://github.com/testowner/testrepo#master-url-cache:
+    - author: Charlie
+      email: charlie@contoso.com
+      files:
+        docs/index.md: |
+          ---
+          author: charlie
+          ---
+          Alice creates this. Bob updates this. Charlie becomes author.
+    - author: Bob 
+      email: bob@contoso.com
+      files:
+        docs/index.md: |
+          Alice creates this. Bob updates this.
+    - author: Alice 
+      email: alice@contoso.com
+      files:
+        docs/index.md: |
+          Alice creates this.
+inputs:
+  docfx.yml: |
```

Comment:
can this be put into repos?

Reply:
The only one cannot be put into repos is `gitCommitsTime` file, otherwise, I cannot consolidate a stable commit id in spec. So I put all the non-content outside repo for consistency.